tests/nix_tests/utils.py

```diff
@@ -540,14 +538,17 @@ def update_node_cmd(path, cmd, i):
 
 
 def update_evmosd_and_setup_stride(modified_bin):
-    def inner(path, base_port, config):
-        update_evmos_bin(modified_bin)(path, base_port, config)
-        setup_stride()(path, base_port, config)
+    def inner(path, base_port):
+        update_evmos_bin(modified_bin)(path)
+        # TODO: correct to remove config here from setup_stride? Because it was not used within
+        setup_stride()(path, base_port)
```

Comment:
I've removed the unused variable for the `config` - not sure if this should have been used instead? @GAtom22

Reply:
we need to keep it to be able to setup stride